integrations.yml

```diff
@@ -354,3 +354,10 @@
   website: "https://www.brightgauge.com/basecamp"
   icon: "brightgauge.png"
   category: 3
+
+-
+  name: "Noko Time Tracking"
+  description: "Fast & friendly time tracking for all your Basecamp projects"
+  website: "https://nokotime.com"
```

Comment:
I've noticed that the logo used for Basecamp in that page is the Basecamp 2 one. Is this integration for Basecamp 3 or Basecamp 2?

Reply:
Hey Rosa!

The integration is for Basecamp Classic, Basecamp 2, and Basecamp 3. Which logo(s) should we use on this page?

Reply:
Cool! If you want, you can use the Basecamp 3 logo, which you can download in different formats and versions from [here](https://basecamp.com/about/press) (scroll down to _Download the Basecamp Logo_).